packages/components/addon/components/hds/dropdown/list-item/copy-item.js

```diff
@@ -4,7 +4,7 @@ import { tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
 
 export default class HdsDropdownListItemCopyItemComponent extends Component {
-  @tracked isSuccess = this.args.isSuccess ?? false;
+  @tracked isSuccess = false;
```

Comment:
[nitpick/not blocker] maybe we could leave it `undefined` by default, so we can use the `false` state for when there is an error.